src/Microsoft.TemplateEngine.TemplateLocalizer/Commands/Export/ExportCommandArgs.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.TemplateEngine.TemplateLocalizer.Commands.Export
+{
+    /// <summary>
+    /// Model class representing the arguments of <see cref="ExportCommand"/>.
+    /// </summary>
+    internal sealed class ExportCommandArgs
```

Comment:
feels very similar to `ExportOptions`.  Any chance for consolidating state?  Even a common ancestor?

Reply:
One of these is used for parsing the command line input, the other one is part of the core API. Some members are common, but they are semantically different. I wouldn't want one of these to change when I modify the other either. So it makes more sense to me having them separate.

Reply:
Common State
      |
      |____________Child_0
      |
      |____________Child_1

Reply:
I already had that picture in mind while writing my comment. Can you elaborate on how that answers my comment?